src/test/java/jenkins/scm/api/SCMEventTest.java

```diff
@@ -209,7 +209,7 @@ public void originOfForwardedRequestMultiHop() throws Exception {
         Mockito.when(req.getHeader("X-Forwarded-Port")).thenReturn("443");
         Mockito.when(req.getHeader("X-Forwarded-For")).thenReturn("scm.example.com, gateway.example.com, proxy.example.com");
         Mockito.when(req.getRequestURI()).thenReturn("/jenkins/notify");
-        Mockito.when(req.getLocalPort()).thenReturn(8080);
+        Mockito.when(req.getRemotePort()).thenReturn(8080);
```

Comment:
That shouldnt be part of this PR, but it is correct. Evaluate the origin of the event was reporting the remote port rather than the local port... only not caught because the tests were using mocks and made the same mistake